src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -46,6 +46,8 @@
 
     private double slackBusPMaxMismatch = SLACK_BUS_P_MAX_MISMATCH_DEFAULT_VALUE;
 
+    private boolean loadMainConnectedComponentOnly = LOAD_MAIN_CONNECTED_COMPONENT_ONLY_DEFAULT_VALUE;
```

Comment:
I am not sure that `load` means something clear for the user, even if it is very clear for the developer. Maybe it could be preferable to use `compute`.

Reply:
Yes I understand, I changed the parameter name.